### PR TITLE
Lesson3 orm homework

### PR DESCRIPTION
**Pull request by vadimzhukov**
Add models Test, Question, Answer, User, Category; add migrations fo NOT NULL constraints to all current models; add migrations for default values for models Test and Answer

### CODE REVIEW

File: db/migrate/20220727130734_create_tests.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateTests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tests do |t|
+      t.string :title
+      t.integer :level
+      t.integer :category_id
```
Comment (BubuntuClu) [inline db/migrate/20220727130734_create_tests.rb:6]: это должен быть внешний ключи. посомтри синтаксис `t.references`

File: db/migrate/20220727130831_create_questions.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateQuestions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :questions do |t|
+      t.string :body
+      t.integer :test_id
```
Comment (BubuntuClu) [inline db/migrate/20220727130831_create_questions.rb:5]: это должен быть внешний ключи.

File: db/migrate/20220727130918_create_answers.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :answers do |t|
+      t.string :body
+      t.integer :question_id
```
Comment (BubuntuClu) [inline db/migrate/20220727130918_create_answers.rb:5]: это должен быть внешний ключи.

**Review by BubuntuClu — CHANGES_REQUESTED** [review]

**Comment by vadimzhukov** [comment]
Переделать
